docs/styles/spec/search-spec.less

```diff
@@ -0,0 +1,48 @@
+@import (reference) "components/search/index";
+
+.search-spec {
+  hx-search-control {
+    /* --------------- Pristine --------------- */
+    > input[type="search"].mock-focus ~ hx-search {
+      .SearchFacade(pristine-focus);
+    }
+
+    /* --------------- Disabled --------------- */
+    > input[type="search"]:disabled {
+      ~ hx-search {
+        .SearchFacade(disabled);
+      }
+
+      &.mock-focus ~ hx-search {
+        .SearchFacade(disabled-focus);
+      }
+    }
+
+    /* --------------- Changed / Touched --------------- */
+    &[hx-dirty] {
+      > input[type="search"]:enabled.mock-focus ~ hx-search {
+        .SearchFacade(pristine-focus);
+      }
+
+      > input[type="search"]:invalid.mock-focus ~ hx-search {
+        .SearchFacade(invalid-focus);
+      }
+    }
+
+    /* --------------- Invalid --------------- */
+    &.hxInvalid > input[type="search"]:enabled.mock-focus {
+      ~ hx-search {
+      .SearchFacade(invalid-focus);
+      }
+
+      ~ button.mock-focus {
+        .SearchFacade(button-focus);
+      }
```

Comment:
I don't think this block is doing anything. It should never match, because you can't focus _both_ the `<input>` _and_ the `<button>`.

Reply:
Block removed.